packages/component/src/Activity/ScrollToEndButton.js

```diff
@@ -19,10 +20,19 @@ const ScrollToEndButton = ({ className }) => {
     focusSendBox();
   }, [focusSendBox, scrollToEnd]);
 
+  const newMessagesText = useLocalize('New messages');
+
   return (
-    <button className={classNames(scrollToEndButtonStyleSet + '', className + '')} onClick={handleClick} type="button">
-      {useLocalize('New messages')}
-    </button>
+    <React.Fragment>
+      <ScreenReaderText text={newMessagesText} />
+      <button
+        className={classNames(scrollToEndButtonStyleSet + '', className + '')}
```

Comment:
Disclaimer: I think you must have a reason.

Why we don't need `aria-live="hidden"` and it didn't announce it twice?